### DISCUSSION TITLE
为什么有些人，一涉及到政治就要关闭思考了？

### DISCUSSION THREAD

**Discussion by richardo876** [WeNeedHome/SummaryOfLoanSuspension#553] [General]
有一些人，一涉及到政治就要关闭自己的思考，而且还要要求别人也要关闭思考。我点进去看他们的账号，有些确实不像是水军。如果某些人真的是热爱并支持共产党的，何不把自己的观点依据说出来和别人辩论？ 非要要求别人不能讨论？

**Comment by b4o** [comment]
现在业主遇到麻烦了，在这扯什么鬼政治，不能改变任何就像苍蝇一样的闻闻有啥用，只会让人更恶心，看热闹不闲事大，是不是BB又给发钱了？？？？

**Reply by richardo876** [reply]
所以到底为什么不能提政治？

**Reply by volans19** [reply]
> 所以到底为什么不能提政治？

因为他们怕一提政治，就被彻底掐断维权的念想了，说到底只在乎自己利益，对问题的源头什么，是谁引发的问题，怎么解决，丝毫不关心
嘴上说着不要提政治，变味了，心里怕的要死，求求了别提政治了，再提没法维权了
只是想跟上边要俩馒头把嘴堵上，万事大吉，而已

说老实话，他们不想谈想当睁眼瞎就当呗，反正自我阉割，阉的是自己伤不到别人，别人帮他骂强奸犯，他说不许你骂强奸我的人，我现在只想把处女膜修好，等待下次强奸，再有什么东西暴雷了，依旧只盯着自己一亩三分地，各扫门前雪就完了，而你党依旧，口头承诺，威逼利诱，分化治之，乐此不疲

**Reply by b4o** [reply]
有用么？除了借机发挥搞混乱能干啥，是能改变天还是能改变地？厕所漏水非说楼盖的地方不对，那放眼地球，更好的在哪里？
就像精美粉？跟着川总搞政变？变成了吗？

**Reply by richardo876** [reply]
所以你是根据哪个事实依据认为是“提政治没用”？  我所看到的事实是：河南储户直骂楼阳生，要人权要法治，不就换来了问题的部分解决了吗？

**Reply by zongjiangU** [reply]
> 所以到底为什么不能提政治？

因为几千年来对官府的恐惧是刻在DNA里的。

**Reply by b4o** [reply]
> 据

hhhhh，原来你理解的政治是这样啊，吵一吵闹一闹，hhhhh那就不说了，你继续。如果有一天你的声音能响彻九霄，那祝贺你

**Reply by Htryhard** [reply]
为什么不能提？你现在连讨论都要搬到github来了，你能在国内某个大平台光明正大讨论烂尾楼为什么会烂尾的问题吗？为什么会烂尾，到底是谁收了钱不办事，不监管，百姓把全身家的钱拿出去买了一个烂尾楼，还不让说了

**Reply by 23lc** [reply]
> 为什么不能提？你现在连讨论都要搬到github来了，你能在国内某个大平台光明正大讨论烂尾楼为什么会烂尾的问题吗？为什么会烂尾，到底是谁收了钱不办事，不监管，百姓把全身家的钱拿出去买了一个烂尾楼，还不让说了

你就事论事谈谁收了钱不办事、不监管，我没意见。
**请具体到主管职能部门。**
讨论这么多条，房管局鲜有人提，各个职能部门之间的矛盾没人分析，有的房企目前为止完全没暴雷也没人总结，这就是你们的思考？

**Reply by 23lc** [reply]
> 所以你是根据哪个事实依据认为是“提政治没用”？ 我所看到的事实是：河南储户直骂楼阳生，要人权要法治，不就换来了问题的部分解决了吗？

有没有一种可能，本来不骂楼阳生就可以在不挨白衣天使打的情况下换来问题的部分解决？
官僚们是怕你指名道姓地骂，还是怕群体性事件？
你吃了带糖衣的药治好了病，能简单归因成只要吃糖衣就能治好病了吗？

**Reply by Htryhard** [reply]
> > 为什么不能提？你现在连讨论都要搬到github来了，你能在国内某个大平台光明正大讨论烂尾楼为什么会烂尾的问题吗？为什么会烂尾，到底是谁收了钱不办事，不监管，百姓把全身家的钱拿出去买了一个烂尾楼，还不让说了
> 
> 你就事论事谈谁收了钱不办事、不监管，我没意见。 **请具体到主管职能部门。** 讨论这么多条，房管局鲜有人提，各个职能部门之间的矛盾没人分析，有的房企目前为止完全没暴雷也没人总结，这就是你们的思考？

谈到具体部门你又没意见了？现在只是说买房这么大的事法律都不完善而已，他不管你，还不让你说话。谈到具体部门就不是政治了？你的观点真是挺奇特的

**Reply by JusticeEER** [reply]
> 有用么？除了借机发挥搞混乱能干啥，是能改变天还是能改变地？厕所漏水非说楼盖的地方不对，那放眼地球，更好的在哪里？ 就像精美粉？跟着川总搞政变？变成了吗？

在中国你不讲政治你能活吗？验个核酸又要感谢，发射个火箭有是伟大，你在微博说中国的事情与共产党无关试试，傻B不要在这乱带节奏

**Reply by 23lc** [reply]
> 谈到具体部门你又没意见了？现在只是说买房这么大的事法律都不完善而已，他不管你，还不让你说话。谈到具体部门就不是政治了？你的观点真是挺奇特的

有没有一种可能，社会是复杂的，人也是复杂的，本来就不止“支持共产党”或“反对共产党”两个观点。
我不反对谈政治。谈具体的事务，不管是谈职能部门如何缺失主体责任，还是业主们应该怎么团结一致、信息共享，这都是谈政治。
但反思来反思去，结果就是一句“都是共产党的错”“都是习某某的错”，这反思质量是不是也太低了？
这种看起来是在“谈政治”，实际上就是在把水搅浑。
彼之砒霜吾之蜜糖，有的职能部门责任缺位，恰恰可能是另外一个部门的KPI。别觉得不可能，产品、研发、测试还是一个公司的，每天不也是在那儿撕逼？“党内无派千奇百怪”，把一个政党当成铁板一块来支持或者反对，本身就是政治上极其幼稚的想法。

**Reply by yalo-ddaa** [reply]
> > 据
> 
> hhhhh，原来你理解的政治是这样啊，吵一吵闹一闹，hhhhh那就不说了，你继续。如果有一天你的声音能响彻九霄，那祝贺你

政治的本质就是决定社会事务的过程，难不成”闭上嘴听话“就叫政治？

**Reply by Joseph-Wheat-Carryier** [reply]
> > 所以到底为什么不能提政治？
> 
> 因为他们怕一提政治，就被彻底掐断维权的念想了，说到底只在乎自己利益，对问题的源头什么，是谁引发的问题，怎么解决，丝毫不关心 嘴上说着不要提政治，变味了，心里怕的要死，求求了别提政治了，再提没法维权了 只是想跟上边要俩馒头把嘴堵上，万事大吉，而已

说中国人有奴性，这就是典型的奴性思维。
天天渴望青天大老爷救世主，哪有救世主来就你？期望别人救你，两腿一跪，哭爹喊娘，官老爷稍一施舍立马磕头谢恩。只关心自己的利益，从来不考虑公众利益。
我看只要一天不改这奴性思维，永远就在中共的黑暗统治下煎熬吧

**Reply by blisee** [reply]
你是哪国人，你祖宗是谁

**Reply by kennard520** [reply]
我们祖宗都不是共产党

**Reply by zongjiangU** [reply]
> 有用么？除了借机发挥搞混乱能干啥，是能改变天还是能改变地？厕所漏水非说楼盖的地方不对，那放眼地球，更好的在哪里？ 就像精美粉？跟着川总搞政变？变成了吗？

**看到这种言论真的很恼火，真的，从心底里恼火，为什么会变成这样！**
+ 讨论政治到底有没有用，能不能改变政策方向，暂且不说，这事就是没用，为什么就不能讨论了，现实里干的没有意义的事情都还少吗？
+ 搞混乱是怎么界定的，不同的声音就是混乱吗，还是说担心现实中发生暴乱？在中国发生零元购，你自己问问自己相信吗？
+ 别动不动扯上美国，这吐槽国民不敢讨论政治的帖子，对美国只字未提，也没半点隐晦的比较，扯美国干嘛。认为党做的不对就是精美， 自己想想这是人话吗，自己的立场到底是中国普通公民还是既得利益者，要是后者，当我放屁好了。
+ 好比是竞技体育，国足亚洲杯小组都不能出线，还不让骂两句了？骂两句就说我是勾结巴西队来瓦解国足内部大团结？犯得着吗我？
**出不了线，立马下课走人！！！！**

**Comment by zxcs001** [comment]
否定自己一辈子以来的习惯很难的，只有真的被铁拳砸了，而且翻不过身的时候，才会真正的审视自己之前的观点的。

大家都一样的，都是人，没有什么特殊地方，美国也是一样的。

你让德州红脖子思考一下，他们那些传统观点是否真的是铁律的时候，人家也不会搭理你的。

不过有一点，红脖子碰了个好国家，被打脸的几率很小，至少他们从祖上能到他们这代能一直愚昧下去。

国内的粉红这几年估计都要被砸个遍了。

**Comment by wedxz** [comment]
> 有一些人，一涉及到政治就要关闭自己的思考，而且还要要求别人也要关闭思考。我点进去看他们的账号，有些确实不像是水军。如果某些人真的是热爱并支持共产党的，何不把自己的观点依据说出来和别人辩论？ 非要要求别人不能讨论？

动上层人的利益，如动他的性命。 动底层人的观念，如动他的祖坟。 而底层观念恰好，正是上层利益的来源。任何揭露这个事实的人，就是二者共同的敌人

**Reply by zcaaron** [reply]
说的不错

**Reply by Murphy0Alan** [reply]
没办法，嘴巴被捂习惯了...

**Reply by NintendoLink** [reply]
🙏

**Comment by lpdll** [comment]
总感觉现在由以前的一个家族专治变成了几个家族一起，啥时候改成xx帝国都不奇怪

**Comment by MicroCountry** [comment]
说的都是道理，现实有那么容易？能别幼稚吗？你认为对共产党有信仰的人就那么傻？共产党和思想是花了几十年，几代人一起打造出来的。并不是现在臭鱼烂虾就能否定的。只要有坚定的人存在，现在的不理想，不纯洁都是暂时的，迟早会出现我们心中的那样

**Reply by b4o** [reply]
它们就是一帮拿了5毛的1450，发的文字基本都是要反天反地的，要是真的这种人当道只会比魔鬼更恶，没有理智，不能独立思考，一出生就在反着反那，为“自由”斗争至到躯体死亡灵魂发臭

**Reply by flame618** [reply]
那么在保留共产党政权的情况下，公民督监督政府，促进下政府进行体制改革有问题吗？

**Reply by zhw173** [reply]
历史是个圈，下一个圈也不可能出现你心中那样

**Reply by ineedabitchlikeu** [reply]
哪有什么主义 都是生意

**Reply by hrsgn** [reply]
拥有这么强大的思想，为什么要控制出版、新闻、言论？不敢放开外网？

**Reply by allenaij** [reply]
是啊，现在的不理想，不纯洁都是暂时的，等共产党自己改良吧，或许要改到下辈子才能实现。

**Comment by TALK1949** [comment]
我来解释一下很多国人眼中所谓的“政治”，既对本国政府不利的皆为“政治”，对本国政府有利的，包括发表不利于他国政府的言论在他们眼里都不是“政治”，可以随便说。
所以，国内人定义的“政治”本身就是“政治”，不谈就是国内人的“政治”正确，自欺欺人，认为说这些没有用只是借口。
再来说一下，政治议题跟帮助业主并不矛盾，因为了解的人都应该明白，只有弄明白自己的处境才能够更好地与公权力周旋，这就是为什么业主要联合起来的原因，因为他们任何人单打独斗，都没有任何意义。
我希望那些在提及政治就不高兴的人，从自己做起，不要浪费大家的流量，因为你们发表的东西才真的是帮不到业主任何的忙，无论是为什么你会出现在这里，以及这个统计为何被搬到了github上，都跟政治有关，所以还是那句话，请不要自欺欺人，如果你不想谈论，先从自己做起。

**Comment by dongyc519** [comment]
因为大家都害怕，害怕因言获罪。另外从心理角度讲，自己不能接受的，也不愿意听到或见到别人能接受。

**Comment by radiateboy** [comment]
不是关闭思考，就事论事。 这里都是业主被坑，一定扯到政治上面？ 觉得中国不好，就去国外呗。 （或者你有能力就推翻他） 只要能把业主问题解决都行。 预售制、购房资金被挪用，西方国家也有这样的例子，就看怎么健全法律保护业主利益了。政治能解决问题，就不要买房子，去搞政治就好了。反正也没体验过资本主义，出生就是特色社会主义。 有能耐去把天换了，把业主问题解决了。我啥话不说，全力支持你。

**Reply by richardo876** [reply]
所以“业主被坑”难道跟“政治”没关系吗？

如果有关系，为啥不能提？   
如果没关系，政治到底有什么用？

**Reply by ppwq2022** [reply]
和ZZ有个屁关系呢，无非就是贪腐，哪个国家没有呢，没脑子的就喜欢上跳下窜

**Reply by richardo876** [reply]
所以你点观点是：政治管不到金融犯罪，也管不到贪腐？

**Reply by radiateboy** [reply]
@ekjhrgkas 你能帮业主把问题解决了，你说什么都是对的。我都支持你。

**Reply by allenyan513** [reply]
醒一醒

**Reply by wedxz** [reply]
觉得中国不好，就去国外呗 == 去了国外就问题不存在了? 什么傻逼逻辑.
出现今天这样的局面不都是在他们英明领导下发生的么,出现问题拿开发商当替罪羊,我要是开发商我也躺平,爱咋咋地

**Reply by yutong16** [reply]
特殊社会主义===》专制资本主义

别人提两句ZZ，你就跳出来高潮，甩风凉话。提ZZ的目的是期望从根本上改良，从源头解决。目前这些问题的出现，不就是权商勾结，操作暗箱化导致的吗？况且百姓还没有渠道监督公权力（专制的恶果）
谈ZZ是大家看到这个问题的本质，或许能引导更多的人清楚。

现今维权行动上，还是要靠百姓向ZF争取。而你在这里喷别人谈ZZ，你又做了什么有意义的事情？

**Reply by radiateboy** [reply]
@yutong16 你能帮业主把问题解决了，你说什么都是对的。我都支持你。

**Reply by JusticeEER** [reply]
西方发达国家没有哪个说他的国家就一定要由他的党领导吧？中国是共产党领导的，出了事不找领导找谁？

**Reply by flame618** [reply]
假设业主被坑扯不到政治上，拼命限制言论自由，把人逼到只能在github上统计烂尾楼数据算不算政治？给储户和烂尾楼业主赋红码算不算政治？没说非要推翻它，能不能多监督它改良它？促进公民权力意识觉醒，促进体制改革，这不是政治的一部分吗？现在的政治局势我相信只要是个懂政治的人都比较悲观，你们不了解政治就不说了，但是连别人懂的人谈一谈都怕得要死，他们只能越来越猖狂，权力越来越集中越来越不受限制，一旦经济增长放缓或者停滞，这种恶性事件只会越来越多，等着瞧吧。

**Reply by Ningmeng-yp** [reply]
现在还有允许思考的地方吗？国内言论不是限制的死死的，但凡负面的消息瞬间摁死，有和别人思考沟通的机会也不至于跑到git上来发言了。

**Reply by perters** [reply]
你说这话就如同天道里讲的 你心里的那个民主其实就是为你做主的那个主。

**Reply by 64xi** [reply]
> 

政治两个字都不敢打出来吗 出了墙还喜欢自我阉割

**Reply by allenaij** [reply]
境外敌对势力又不能帮业主挽回损失，只能寄希望于共产党。
万一共产党被推翻了，新政府又不管，业主可怎么办啊！

**Comment by BobbyYang9** [comment]
小孩子大部分是不觉得被爸爸大屁股是丢人的行为，因为关起门来打，最多哭两下，跟父亲讨个饶就结束了。小孩子觉得丢人是因为爸爸当着大街上把他/她裤子拖了，打屁股，这个时候孩子心里不光是痛苦，更多的是对父亲的深深的恨意。所以羞不羞愧不在于大屁股这个行为本身，而在于有没有被别人看到。政治的“先进性”好比孩子心中父亲所有的正面形象，也似乎是唯一的光辉，如果连先进性都没了，那就彻底崩塌。现在孩子在面对家暴父亲的态度是很微妙的，一方面有了深深的恨意，一方面又不想把父亲报警抓起来，怕自己成了孤儿还被邻居指指点点，说这小子大义灭亲。恰好这个痛点父亲一开始就知道，于是乎，在一遍又一遍的测试后就肆无忌惮的打了。因为打的太放肆了，很可能哪天打傻了，打死了，才会被抓起来吧。

**Comment by z-huabao** [comment]
我建议不要在这个谈，政治太泛了，谈起来话题就分散了，这样就不能集中力量到烂尾楼的事件中来！

毕竟你不能指望一次性解决所有问题，所以在这里讨论国内政治等同于捣乱。

**Reply by richardo876** [reply]
部分同意你的观点。当然最好不要分散问题，但要谈论烂尾楼事件就不能避开烂尾楼所涉及的政治问题。

**Reply by hrsgn** [reply]
你以为统战价值是怎么来的？

**Comment by hoootWind** [comment]
1. 这个帖子说的很清楚，“这里不做任何非数据性质等其他一系列讨论，只是统计一个数据，有错就纠改，无其他任何含义”
2. 业主诉求是处理好烂尾楼的问题，不是意识形态的问题
3. 没有人涉及到政治就关闭思考，也不是众人皆醉你独醒，只是想与不想讨论而已
4. 你想讨论政治，完全可以，这并没有什么，可以自己开一个帖子好好讨论，也可以找个开阔处、繁华处甚至有关单位里讨论。你在他人的帖子下违背他人的意愿，反而又攻击不愿讨论的人，这是不是不太合适？

**Reply by TALK1949** [reply]
1，请先从自己做起。
2，请先从自己做起。
3，请先从自己做起。
4、既然政治可以不是政治，那么这里的讨论自然就和政治没有关系，所以为什么你要扯政治呢？

结论：拒绝政治讨论，请先从自己做起。😂

**Comment by tencentbook** [comment]
政治就是把自己的朋友变多，把自己的敌人变少。

**Comment by KisaraKira** [comment]
有的是跪久了站不起来的，有的是监狱的网评员在努力减刑。

**Comment by wangyue57** [comment]
中国房市这么多年了，炒房第一主力：地方政府。第二主力，体制内稳定收入群体。

地方政府卖地，卖地的钱给公务员发工资发超额福利，公务员拿钱买房，左脚踩右脚，直接升天。直到烂尾之前，公务员和房企一直是赚的，只有屁民亏。房企资金链断裂，政府限跌不让房企促销回血，是为什么？  公务员买房赚的钱，不想吐出来呗。 房企可以死，公务员赚的钱得锁死了。

房子就是政治，提房子不提政治，搁这装外宾呢？

**Reply by zongjiangU** [reply]
有道理。
不过觉得有一点可能稍微片面
>  房企资金链断裂，政府限跌不让房企促销回血，是为什么？ 公务员买房赚的钱，不想吐出来呗.
绝大多数公务员也是韭菜，限跌就是怕出现大规模断贷，小银行计提这么大的坏账可能就撑不住了，一旦开始出现银行破产，金融信用降低会形成挤兑，大银行流动性抽干，也就倒下了。这样的金融灾难，会毁掉半数以上的家庭，自然的政府公信力也会下降，政权就不稳了。

**Reply by BuryMe** [reply]
94年分税改革，地方政府税收大头上交了，没钱发展，这个时候发现了土地这个好东西，于是诞生了以卖地为核心的土地财政。但是土地财政的收入还是杯水车薪，学校，医院，高架。。。根本不够，于是地方政府开始向银行借钱发展，到这里，土地财政升级为了土地金融，核心更是要狠狠地卖，高价卖，银行利息是要还的。银行借给政府的钱怎么来的？印钞机印出来的，银行为啥敢印？因为地方政府抵押了土地。。。。。轰轰烈烈20年，最终一个结果，土地成了人命币的锚！房价就是支撑人命币的核心！楼市就是国运！

所以，房子不止是政治，房子就是国运！

**Comment by w814929152** [comment]
真正的共产党人不屑于隐藏自己的观点，那些自己不说话，也不让别人说话的人肯定不是共产党人就是了。

**Comment by xiejun946** [comment]
妈的，天天谈政治，来实名制发言，在说谈政治这个事

**Reply by wedxz** [reply]
这是在国内被锤出幻觉了吗?github不捶你都不适应了?

**Comment by Fakeytakk** [comment]
我个人的观点是在这个帖讨论更深层次的问题，负责人容易被“负责”，所以为了能够保全自身，还是当心比较好。
至于楼里说政治和贪腐，贪腐就是政治的一部分，同时政治就是生活，政治影响着生活的方方面面，例如绿码。你撇不开，甩不掉。

**Comment by LeDevv** [comment]
建议那些对政治有臆想的人看看这本书：
https://zh.jp1lib.org/book/17246524/2ef236

**Comment by gaozongy** [comment]
建立这个仓库的目的是什么? 是让哪些拿不到房的业主能够争取到自己的利益, 让开发商顺利交房. 而不是你们要推翻这个推翻那个, 再说了, 推翻了就更好了吗? 哪些动不动就扯国家, 扯共产党的人, 你们无非就是找到理由发泄自己的不满, 看热闹不嫌事大, 对那些倾家荡产, 拿不到房子的人有个JB用!

**Reply by hrsgn** [reply]
批评政府=推翻？顺民配有统战价值？

**Reply by allenaij** [reply]
现在要依靠的就是党和国家，万一党和国家被这些敌对势力弄垮了，新政府可不会解决咱们业主的烂尾楼！

**Reply by TALK1949** [reply]
> 现在要依靠的就是党和国家，万一党和国家被这些敌对势力弄垮了，新政府可不会解决咱们业主的烂尾楼！

你连新政府能不能解决问题都知道了，可否问一下是谁怎么推翻的当前政权？
对着稻草人无限上纲很有意思？我怎么看你倒挺像是“境外势力”。

**Reply by allenaij** [reply]
> > 现在要依靠的就是党和国家，万一党和国家被这些敌对势力弄垮了，新政府可不会解决咱们业主的烂尾楼！
> 
> 你连新政府能不能解决问题都知道了，可否问一下是谁怎么推翻的当前政权？ 对着稻草人无限上纲很有意思？我怎么看你倒挺像是“境外势力”。

咱们业主现在就是要抓住共产党这根救命稻草，万一共产党没了，那就没有指望了。
难道让另外一个党还是民运组织接盘烂尾楼？不信你组个党试试。

**Reply by TALK1949** [reply]
> > > 现在要依靠的就是党和国家，万一党和国家被这些敌对势力弄垮了，新政府可不会解决咱们业主的烂尾楼！
> > 
> > 
> > 你连新政府能不能解决问题都知道了，可否问一下是谁怎么推翻的当前政权？ 对着稻草人无限上纲很有意思？我怎么看你倒挺像是“境外势力”。
> 
> 咱们业主现在就是要抓住共产党这根救命稻草，万一共产党没了，那就没有指望了。 难道让另外一个党还是民运组织接盘烂尾楼？不信你组个党试试。

好感人的逻辑，这跟共产党没了有啥关系？
说的好像这些烂尾楼都不是在贵党治下出现的一样，有些楼都烂尾十多年了，也没见贵党解决，要不是业主组织起来集体停贷他们才懒得鸟你，按您这逻辑要是共产党解决不了你的问题，您是不是就考虑要造反了？嗯，我看你越来越像“境外势力”了。😂

**Reply by allenaij** [reply]
> > > > 现在要依靠的就是党和国家，万一党和国家被这些敌对势力弄垮了，新政府可不会解决咱们业主的烂尾楼！
> > > 
> > > 
> > > 你连新政府能不能解决问题都知道了，可否问一下是谁怎么推翻的当前政权？ 对着稻草人无限上纲很有意思？我怎么看你倒挺像是“境外势力”。
> > 
> > 
> > 咱们业主现在就是要抓住共产党这根救命稻草，万一共产党没了，那就没有指望了。 难道让另外一个党还是民运组织接盘烂尾楼？不信你组个党试试。
> 
> 好感人的逻辑，这跟共产党没了有啥关系？ 说的好像这些烂尾楼都不是在贵党治下出现的一样，有些楼都烂尾十多年了，也没见贵党解决，要不是业主组织起来集体停贷他们才懒得鸟你，按您这逻辑要是共产党解决不了你的问题，您是不是就考虑要造反了？嗯，我看你越来越像“境外势力”了。😂

家里有人在事业单位工作，别说造反，就是去现场举个牌，家人也被施压。有的人靠国家养着，有的人必须自己养活自己，这不一样。

**Reply by TALK1949** [reply]
> 家里有人在事业单位工作，别说造反，就是去现场举个牌，家人也被施压。有的人靠国家养着，有的人必须自己养活自己，这不一样。

一直扯造反的不是你？别装了，我看你就是境外势力，说，你有什么目的！😂

**Comment by syyxsxx** [comment]
谈政治可以去建政论坛，这里没必要谈政治

**Comment by phxD** [comment]
因为他们只关心自己的利益，所以利益受损就会跳起来骂人，所以只会继续被当两脚羊
